### ISSUE TITLE
Unify the tran

### ISSUE THREAD

**Issue by PeterDaveHello** [anaclumos/heimdall#70]
Currently, the same term may or may not be translated (depending on the terminology) into another language using different terms in the title and the rest of the content.

Maybe GPT-4o should process the title, summary, reaction, and translation in the same step or prompt to ensure consistency in the translated result. What do you think?

**Comment by anaclumos** [comment]
I agree. Investigating to incorporate braintrust.dev for experimenting things

**Comment by PeterDaveHello** [comment]
Cool, I haven't used a tool like that. I initially thought we could do it just at the code level, but I don't know which is better.

**Comment by PeterDaveHello** [comment]
@anaclumos I initially used GitHub's Copilot Workspace to generate some changes. Would that be something you'd like to take a look at? Or should we just forget it and wait for the new implementation?

**Comment by PeterDaveHello** [comment]
> Could you send me the implementation details?

I can use Copilot Workspace just to push the changes to a single branch for your review, even better, pull requests can also be created if you wanted 🤩

**Comment by PeterDaveHello** [comment]
All yours 🎉 

- https://github.com/PeterDaveHello/heimdall/tree/unify-translation
- https://github.com/PeterDaveHello/heimdall/tree/unify-translation-another-ver

I basically did nothing but ask AI to determinate what to do, so from my experience, they both may not work without additional modification.

**Comment by anaclumos** [comment]
@PeterDaveHello fyi this will be wip in https://github.com/anaclumos/signalkite/tree/signalkite branch. I'm a bit slow cuz of my day job, but good works are in cooks. as you might have noticed, i eventually decided to even adopt monorepos to create a dedicated workspace for prompt engineering
